lib/services.identity.js

```diff
@@ -376,6 +432,60 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
       404: 'The identity was not found on the system.'
     }
   });
+
+  app.delete(idPath, ensureAuthenticated, function(req, res, next) {
+    var fullUrl = req.protocol + '://' + req.get('host') + req.originalUrl;
+    brIdentity.setStatus(req.user.identity, fullUrl, 'deleted', function(err) {
+      if(err) {
+        return next(err);
+      }
+      res.sendStatus(204);
+    });
+  });
+
+  // TODO: endpoint for batch update operations, needs fix-up and tests
+  app.patch(basePath,
+    ensureAuthenticated,
+    function(req, res, next) {
+      var operations = req.body;
+      var deleteOperations = operations.filter(function(operation) {
+        return operation.op === 'delete';
+      });
+      var updateOperations = operations.filter(function(operation) {
+        return operation.op === 'update';
+      });
+      var updateRoleOperations = operations.filter(function(operation) {
+        return operation.op === 'updateRoles';
+      });
+
+      async.auto({
+        delete: function(callback) {
+          return callback();
+        },
+        update: function(callback) {
+          return callback();
+        },
+        updateRoles: function(callback) {
+          if(updateRoleOperations.length === 0) {
+            return callback();
+          }
+          async.each(updateRoleOperations,
+            function(operation, callback) {
+              var identity = operation.identity;
+              brIdentity.setRoles(null, identity, function(err, record) {
+                return callback(err, record);
+              });
+            }, function(err, results) {
+              return callback(err, results);
+            });
+        }
+      }, function(err, results) {
+        if(err) {
+          return next(err);
+        }
+        res.json(results);
```

Comment:
We should not be returning `results` from the async API to the client. Maybe a 204 instead?